src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -154,19 +154,9 @@ public function getRawImportMapData(): array
 
         $rawImportMapData = [];
         foreach ($allEntries as $entry) {
-            if ($entry->path) {
-                $asset = $this->findAsset($entry->path);
-
-                if (!$asset) {
-                    throw new \InvalidArgumentException(sprintf('The asset "%s" cannot be found in any asset map paths.', $entry->path));
-                }
-            } else {
-                $sourcePath = $this->packageDownloader->getDownloadedPath($entry->importName);
-                $asset = $this->assetMapper->getAssetFromSourcePath($sourcePath);
-
-                if (!$asset) {
-                    throw new \InvalidArgumentException(sprintf('The "%s" vendor asset is missing. Run "php bin/console importmap:install".', $entry->importName));
-                }
+            $asset = $this->findAsset($entry->path);
+            if (!$asset) {
+                throw $this->createMissingImportMapAssetException($entry);
```

Comment:
Example of why it's nice now that `ImportMapEntry.path` is populated even for vendor/remote assets :)